design/assets/visualidentity2025.md

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
create a new directory called brand-2025, move this file to that directory and call it PLANNING.md